phpstan-baseline.neon

```diff
@@ -0,0 +1,50 @@
+parameters:
+	ignoreErrors:
+			-
+			    message: "#^Parameter \\#1 \\$product of method Pimcore\\\\Bundle\\\\EcommerceFrameworkBundle\\\\OfferTool\\\\AbstractOfferItem\\:\\:setProduct\\(\\)\\ expects Pimcore\\\\Model\\\\Element\\\\AbstractElement\\|null, Pimcore\\\\Bundle\\\\EcommerceFrameworkBundle\\\\Model\\\\CheckoutableInterface\\ given\\.$#"
+			    reportUnmatched: false
+			    count: 2
+			    path: src/OfferTool/DefaultService.php
+
+			-
+			    message: "#^Parameter \\#1 \\$product of method Pimcore\\\\Bundle\\\\EcommerceFrameworkBundle\\\\Model\\\\AbstractOrderItem\\:\\:setProduct\\(\\)\\ expects Pimcore\\\\Model\\\\Element\\\\AbstractElement\\|null, Pimcore\\\\Bundle\\\\EcommerceFrameworkBundle\\\\Model\\\\CheckoutableInterface\\ given\\.$#"
+			    reportUnmatched: false
+			    count: 1
+			    path: src/OrderManager/V7/OrderManager.php
+
+			-
+			    message: "#^Parameter \\#1 \\$products of method Pimcore\\\\Bundle\\\\EcommerceFrameworkBundle\\\\PricingManager\\\\Condition\\\\CatalogProduct\\:\\:setProducts\\(\\)\\ expects array\\<Pimcore\\\\Bundle\\\\EcommerceFrameworkBundle\\\\Model\\\\AbstractProduct\\>, array\\<int<0, max>, Pimcore\\\\Model\\\\DataObject\\\\Concrete\\> given\\.$#"
+			    reportUnmatched: false
+			    count: 1
+			    path: src/PricingManager/Condition/CatalogProduct.php
+
+			-
+			    message: "#^Parameter \\#1 \\$product of method Pimcore\\\\Bundle\\\\EcommerceFrameworkBundle\\\\PricingManager\\\\EnvironmentInterface\\:\\:setProduct\\(\\)\\ expects Pimcore\\\\Bundle\\\\EcommerceFrameworkBundle\\\\Model\\\\CheckoutableInterface\\, null given\\.$#"
+			    reportUnmatched: false
+			    count: 1
+			    path: src/PricingManager/PricingManager.php
```

Comment:
Would be nice to fix this 
https://github.com/pimcore/ecommerce-framework-bundle/blob/785d295ed2fc98c0c74b6ee32afa194628a652a5/src/PricingManager/PricingManager.php#L131

seem should be nullable
https://github.com/pimcore/ecommerce-framework-bundle/blob/785d295ed2fc98c0c74b6ee32afa194628a652a5/src/PricingManager/Environment.php#L78

https://github.com/pimcore/ecommerce-framework-bundle/blob/785d295ed2fc98c0c74b6ee32afa194628a652a5/src/PricingManager/EnvironmentInterface.php#L48